notebooks/en/fine_tuning_llm_for_generate_persian_product_catalogs_in_json_format.ipynb

```diff
@@ -0,0 +1,603 @@
+{
```

Comment:
<p>you can move this up ⬆️ above the chat template, also we are using I and we interchangeably, best to stick to we.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/86/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>